src/GovUk.Education.ExploreEducationStatistics.Content.Model/DataBlockSummary.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Unused import?

Reply:
It is being used. Seems like nothing has changed in this file, but github is suggesting this line changed?

Either way, this file is very likely to be removed in the follow-up cleanup.